src/lib/service/bootstrap.ts

```diff
@@ -0,0 +1,33 @@
+import { DataContext } from 'lib/context/crudContext';
+import { Omit } from 'lib/helpers';
+import { Definitions } from 'lib/settings/definitions';
+import { CrudRepository } from 'lib/settings/repository';
+
+const createDefaultImplementation = <T, C>(): Omit<Required<Definitions<T>>, 'repository'> => ({
+    detail: () => Promise.reject(new Error('"detail" not implemented')),
```

Comment:
It suprises me that a fan of FP like you does not use `constant` functions to represent these

Reply:
There is no FP / utility library used in Crudella. Using custom `constant` or `always` is possible here, but since its just few lines in one block, I dare say it would not improve legibility (defining custom fuction for this fart). Only elegance somewhat.